### ISSUE TITLE
@bugsnag/plugin-react unmet peer dependency

### ISSUE THREAD

**Issue by msteward** [bugsnag/bugsnag-js#438]
Installing the new v5 of Bugsnag by following the instructions on https://docs.bugsnag.com/platforms/javascript/react/ results in the following warning on the console:

```
warning " > @bugsnag/plugin-react@5.0.0" has unmet peer dependency "@bugsnag/browser@* || >=5.0.0-rc.1".
```

It's only a warning but suggest either making @bugsnag/browser a dependency or add it to the installation instructions if it's actually required.

**Comment by msteward** [comment]
@bengourley I'm using yarn 1.12.3

I was installing it alongside `@bugsnag/js` but it still throws the warning about `@bugsnag/browser`. Note this doesn't seem to cause any errors but could be confusing for folks.

**Comment by bengourley** [comment]
I agree it could be confusing. Thanks for bringing it up 🙏 

`@bugsnag/browser` _is_ installed as a dependency of `@bugsnag/js`, so it shouldn't complain after it's resolved everything. I'll get this looked into.

**Comment by snmaynard** [comment]
I imagine it doesnt pick up transitive dependencies so you'd have to have a peer on js.

**Comment by evgmel** [comment]
I have this versions and packages at my package.json dependencies section:
```
"@bugsnag/js": "^5.1.0",
"@bugsnag/plugin-vue": "^5.1.0",
```
And I have the same warning:

> [3/4] Linking dependencies...
> warning " > @bugsnag/plugin-vue@5.1.0" has unmet peer dependency "@bugsnag/browser@* || >=5.0.0-rc.1".

I'm using Yarn 1.12.3

**Comment by anikolaev** [comment]
@bengourley I am reinstalling it right now and it still installs `@bugsnag/plugin-react@6.0.0-alpha.1`
```
Direct dependencies
  @bugsnag/js@5.2.0
  @bugsnag/plugin-react@6.0.0-alpha.1
```

**Comment by bengourley** [comment]
Ah, apologies! I've updated the dist-tags for the plugins now too.

**Comment by anikolaev** [comment]
Great, it installs 5.2.0 now. But the original issue is still reproducable:

`warning " > @bugsnag/plugin-react@5.2.0" has unmet peer dependency "@bugsnag/browser@* || >=5.0.0-rc.1".`

**Comment by Santas** [comment]
This also happens for express. We have
```
    "@bugsnag/js": "^5.2.0",
    "@bugsnag/plugin-express": "^5.2.0",
    "@bugsnag/plugin-react": "^5.2.0",
```
and get on `yarn install`:
```
warning " > @bugsnag/plugin-express@5.2.0" has unmet peer dependency "@bugsnag/node@* || >=5.0.0-alpha.0".
warning " > @bugsnag/plugin-react@5.2.0" has unmet peer dependency "@bugsnag/browser@* || >=5.0.0-rc.1".
```

**Comment by bengourley** [comment]
Yeah it's a difference in behaviour between `yarn` and `npm`. It will be the same for any of our plugins currently, as the dependency tree looks like this:

```
@bugsnag/js
├── @bugsnag/node
└── @bugsnag/browser
```

```
@bugsnag/react
└── @bugsnag/browser (peer dep)
```

npm resolves the tree and then emits any warnings (so is silent, because the peer dependency is satisfied) but yarn does it from looking either at the manifest or just the top level dependencies.

The solution is to make the plugins just depend on the "universal" shim package `@bugsnag/js` rather than the platform specific notifier and we'll get to it at some point in the near future!

**Comment by bengourley** [comment]
This was fixed in #480 and will go out with `v6.0.0` in the next few weeks.
